_utility/send_gh_comment_validation.sh

```diff
@@ -0,0 +1,105 @@
+# Define help function
+function help(){
```

Comment:
The functions that do not belong directly to this file can be placed elsewhere and we can source it in, so they are still visible to this file - we can pair on this if you like.